### PR TITLE
Replace - Inputs & buttons from blade-components 

### PR DESCRIPTION
**Pull request by Roene-JustBetter**
- I have replaced the old inputs & buttons for the inputs & buttons from the blade-components.
- Make use of the new colors like `text-muted` or `bg`
- I have checked and removed text-color where it is not needed because the body has a class `text` so we don't need to define the default text-color everywhere. 
- I have added missing Dutch translations from this package -- also checked that we don't have duplicate translations from the Rapidez/core. 

See some screenshot below:
Scherm­afbeelding 2024-12-06 om 15 30 27 (3)
Scherm­afbeelding 2024-12-06 om 15 30 56 (3)
Scherm­afbeelding 2024-12-06 om 15 31 12 (3)

_We only need this in V3_

### CODE REVIEW

**Comment by royduin** [comment]
Waiting for at least a good review from @jordythevulder or @JimmyHoenderdaal

**Review by jordythevulder — APPROVED** [review]

**Pull request merged**
